### ISSUE TITLE
compile errors

### ISSUE THREAD

**Issue by WheresWally** [bportaluri/WiFiEsp#18]
From the latest version, i get this list of complaints from the compiler
the previously downloaded version will compile ok
Terry

C:\Users\terry\AppData\Local\Temp\build6429296284531538672.tmp\ECU_yes2.cpp.o 
ECU_yes2.ino: In function 'void SetUpGETRealTimeData()':
ECU_yes2.ino:128:26: error: no matching function for call to 'WiFiEspClient::print(const char [7])'
ECU_yes2.ino:128:26: note: candidate is:
In file included from C:\Program Files (x86)\Arduino\libraries\WiFiEsp-master\src/WiFiEsp.h:28:0,
                 from ECU_yes2.ino:14:
C:\Program Files (x86)\Arduino\libraries\WiFiEsp-master\src/WiFiEspClient.h:39:10: note: size_t WiFiEspClient::print(const __FlashStringHelper*)
   size_t print(const __FlashStringHelper *ifsh);
          ^
C:\Program Files (x86)\Arduino\libraries\WiFiEsp-master\src/WiFiEspClient.h:39:10: note:   no known conversion for argument 1 from 'const char [7]' to 'const __FlashStringHelper*'
ECU_yes2.ino: In function 'void SetUpGETHomeData()':
ECU_yes2.ino:143:26: error: no matching function for call to 'WiFiEspClient::print(const char [7])'
ECU_yes2.ino:143:26: note: candidate is:
In file included from C:\Program Files (x86)\Arduino\libraries\WiFiEsp-master\src/WiFiEsp.h:28:0,
                 from ECU_yes2.ino:14:
C:\Program Files (x86)\Arduino\libraries\WiFiEsp-master\src/WiFiEspClient.h:39:10: note: size_t WiFiEspClient::print(const __FlashStringHelper*)
   size_t print(const __FlashStringHelper *ifsh);
          ^
C:\Program Files (x86)\Arduino\libraries\WiFiEsp-master\src/WiFiEspClient.h:39:10: note:   no known conversion for argument 1 from 'const char [7]' to 'const __FlashStringHelper*'
no matching function for call to 'WiFiEspClient::print(const char [7])'

**Comment by bportaluri** [comment]
What version of Arduino IDE do you have?
I am using the latest one 1.6.7.

**Comment by WheresWally** [comment]
1.6.4

**Comment by WheresWally** [comment]
fixed it by modifying the using clause in WiFiEspClient.h

```
  using Print::write;
  using Print::print;
  using Print::println;

**Comment by bportaluri** [comment]
Fixed in https://github.com/bportaluri/WiFiEsp/commit/8d2b288d307a1eb1373955f73c1f61254e5666b5
Released 1.4.2

Thank you
